CRM/Case/Form/ActivityToCase.php

```diff
@@ -22,11 +22,13 @@ class CRM_Case_Form_ActivityToCase extends CRM_Core_Form {
 
   /**
    * Build all the data structures needed to build the form.
+   *
+   * @throws \CRM_Core_Exception
    */
   public function preProcess() {
     $this->_activityId = CRM_Utils_Request::retrieve('activityId', 'Positive');
     if (!$this->_activityId) {
-      CRM_Core_Error::fatal('required activity id is missing.');
+      CRM_Core_Error::statusBounce('required activity id is missing.');
```

Comment:
They all seem ok I'm just not sure about this one. This form is a form but not a form - as far as I know it's only reached via an ajax snippet, and when I try to force a failure it gives me a weird screen with the error squished inside an unnecessarily small textbox. But I get the same thing before the patch so that's ... good?

Reply:
@demeritcowboy I've switched it to throw an Exception since there is no clear case for a statusBounce and throwing an exception is closer to no change

Reply:
Sounds good!